translate/src/store.ts

```diff
@@ -6,15 +6,18 @@ import history from './historyInstance';
 import createRootReducer from './rootReducer';
 
 const store = configureStore({
+  // @ts-ignore Here be dragons
```

Comment:
This smells like something we should have a follow-up for?

My gut says we have a typing bug here. In the sense of 1+.

Reply:
How about if I file a catch-all issue after this lands for tracking all remaining tsc & eslint error workarounds? Would be easier afterwards, as then could point at specific lines of code on `master`.

Reply:
So, I updated VS Code. Which ships TS 4.6, and this error suddenly is pretty omnipresent. That makes it kinda awkward. Only two errors on master (but still two), one is shared, the other is commentTabRef which I think you're fixing on this branch.

I think the problem is that we don't have an app action type. The reducers just take the local action type, and that doesn't align with the app action type.

Found 46 errors in 15 files.

Errors  Files
     3  src/App.tsx:57
     2  src/core/comments/actions.ts:24
     1  src/core/editor/actions.ts:312
     1  src/core/editor/hooks/useUpdateTranslationStatus.ts:44
     1  src/core/l10n/components/AppLocalizationProvider.tsx:39
     1  src/core/plural/actions.ts:23
     1  src/core/plural/components/PluralSelector.tsx:48
     2  src/modules/batchactions/actions.ts:88
     4  src/modules/batchactions/components/BatchActions.tsx:76
     6  src/modules/entitieslist/components/EntitiesList.tsx:144
    14  src/modules/entitydetails/components/EntityDetails.tsx:135
     2  src/modules/history/actions.ts:185
     3  src/modules/navbar/components/Navigation.tsx:46
     3  src/modules/search/components/SearchBox.tsx:184
     2  src/store.ts:9

Reply:
Ugh. Yeah, I see that too. On top of #2444 these are down to 37 errors in 12 files with TS 4.6.

As a workaround, in VS Code you can set it to use the local TS version: https://code.visualstudio.com/docs/typescript/typescript-compiling#_using-the-workspace-version-of-typescript

Could we punt all of these to be a separate issue?

Reply:
Issue filed: #2453.